src/top_model/main.cpp

```diff
@@ -14,14 +14,19 @@
 #include <cadmium/logger/common_loggers.hpp>
 
 
-#include "../../lib/vendor/NDTime.hpp"
+#include "../../lib/DESTimes/include/NDTime.hpp"
 #include "../../lib/vendor/iestream.hpp"
 
 #include "../../include/data_structures/message.hpp"
 
-#include "../../include/atomics/senderCadmium.hpp"
-#include "../../include/atomics/receiverCadmium.hpp"
-#include "../../include/atomics/subnetCadmium.hpp"
+#include "../../include/atomics/sender_cadmium.hpp"
+#include "../../include/atomics/receiver_cadmium.hpp"
+#include "../../include/atomics/subnet_cadmium.hpp"
+
+
+#define MAIN_OUTPUT "../data/abp_output.txt"
```

Comment:
You have added this but haven't utilized it anywhere. 
My suggestion will be to add the file paths outside the main function so that they are easy to modify. Take a look at this.